<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorPanel</class>
 <widget class="QDialog" name="DoctorPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_FIO">
        <property name="text">
         <string>ФИО</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_age">
        <property name="text">
         <string>Возраст</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_telephone">
        <property name="text">
         <string>Телефон</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_cabinet">
        <property name="text">
         <string>Кабинет</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_post">
        <property name="text">
         <string>Должность</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_totalAppointments">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Текущие записи</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
